src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderService.java

```diff
@@ -1829,7 +1830,7 @@ public Map<String, Object> populatePreviewOrder(String authorisation, CallbackRe
             caseData.setCourtName(callbackRequest
                                       .getCaseDetailsBefore().getData().get(COURT_NAME).toString());
         }
-        if (caseData.getCreateSelectOrderOptions() != null && caseData.getDateOrderMade() != null) {
```

Comment:
Why is it needed? Does this change affects any other orders?

Reply:
This change was earlier there to differentiate between uploaded order and created order, placed a new  logic  as C6 and C6a are not going to have this dateordermade field , as this fields populates from common screen.., tested most of the flow, it looks ok but gonna test few more orders to check for any error.